app/javascript/controllers/pair_requests_form_controller.js

```diff
@@ -0,0 +1,59 @@
+import { Controller } from "@hotwired/stimulus"
+import dayjs from "dayjs"
+import utc from "dayjs/plugin/utc"
+import timezone from "dayjs/plugin/timezone"
+import customParseFormat from "dayjs/plugin/customParseFormat"
+dayjs.extend(utc)
+dayjs.extend(timezone)
+dayjs.extend(customParseFormat)
+
+export default class extends Controller {
+  
+  static targets = ["invitee", "inviteeTz", "inviterRequestTime", "inviteeInfo",]
+  static values = {
+    input : String
+  }
+
+  onClick(event) {
+    if(this.inviterRequestTimeTarget.value && this.inviteeInfoTarget.value !== ''){
```

Comment:
not blocking (I added an issue for this to incorporate a js linter), but there are style inconsistencies that exist in the file. Typically, we want spaces between an `if` and the `(`.

Reply:
I see, will keep that in mind for future reference!